common-lib/src/main/scala/com/gu/mediaservice/model/ThrallMessage.scala

```diff
@@ -0,0 +1,88 @@
+package com.gu.mediaservice.model
+
+ import com.gu.mediaservice.lib.logging.{GridLogging, LogMarker}
+ import com.gu.mediaservice.model.leases.MediaLease
+ import com.gu.mediaservice.model.usage.UsageNotice
+ import org.joda.time.{DateTime, DateTimeZone}
+ import play.api.libs.json.{JodaReads, JodaWrites, Json, __}
+
+sealed trait ThrallMessage extends GridLogging with LogMarker {
+  implicit val yourJodaDateReads = JodaReads.DefaultJodaDateTimeReads.map(d => d.withZone(DateTimeZone.UTC))
+  implicit val yourJodaDateWrites = JodaWrites.JodaDateTimeWrites
+  def subject: () => String = () => this.getClass.getSimpleName
```

Comment:
[minor] could this be a `val` or will that return `ThrallMessage` rather than the name of case class implementation

Reply:
that can safely be a val of `String` and will do what we want.

Reply:
yeah, hadn't realised trait values or those in the body of a case class won't get serialised